inbm/dispatcher-agent/fpm-template/usr/share/dispatcher-agent/manifest_schema.xsd

```diff
@@ -119,7 +119,6 @@
                             <xs:element name="hash_algorithm" minOccurs="0" maxOccurs="1">
                                 <xs:simpleType>
                                     <xs:restriction base="xs:string">
-                                        <xs:enumeration value="256"/>
                                         <xs:enumeration value="384"/>
```

Comment:
This hasn't been used yet by anyone, so it's alright to remove one of the options.  We should support 384 and higher only.